setup.cfg

```
@@ -26,6 +25,11 @@ classifiers =
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: Implementation :: PyPy
     Operating System :: OS Independent
+project_urls =
+    Documentation = https://celery-batches.readthedocs.io
+    Release notes = https://github.com/clokep/celery-batches/blob/main/CHANGELOG.rst
+    Source = https://github.com/clokep/celery-batches
+    Tracker = https://github.com/clokep/celery-batches/issues
```

Looks good

You can also add a donation link, e.g. `https://github.com/sponsors/<username>`

```suggestion
    Tracker = https://github.com/clokep/celery-batches/issues
    Donate = https://github.com/sponsors/clokep
```

---

I unfortunately haven't gotten around to setting GitHub sponsors up yet. On my to do list though! 👍